spec/registrant_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+RSpec.configure do |config|
+  config.formatter = :documentation
+end
+
+RSpec.describe Registrant do
+
+  describe '#initialize' do
+    it 'can initialize' do
+      registrant_1 = Registrant.new('Bruce', 18, true)
+      registrant_2 = Registrant.new('Penny', 16 )
+      registrant_3 = Registrant.new('Tucker', 15 )
+
+      expect(registrant_1).to be_instance_of(Registrant)
```

Comment:
just testing this for one Registrant object is sufficient IMO -- just populating the attributes in the same way (i.e. there's no sad path or edge case to consider) doesn't need to be tested with multiple objects

Reply:
Ahh I see, thanks!